src/v2/Apps/Order/Components/SavedAddresses.tsx

```diff
@@ -177,11 +173,11 @@ const SavedAddresses: React.FC<SavedAddressesProps> = props => {
     }
 
     const isDefaultAddress = address.isDefault
-
     return (
       <BorderBox
         p={2}
-        width="100%"
+        mb={2}
+        width={340}
```

Comment:
(non-blocking) Fixed widths like this don't scale so well when the the browser is at various sizes. Can we use a percent width instead of something fixed? If not, no prob; we just want to be sure that it works in different browsers and on mobile web.